### ISSUE TITLE
Footer in user manual should report last modified date

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor.org#130]
The footer in the user manual HTML document should report the last modified date instead of the last date of publish.

Ideally, the date string should link to the document history on GitHub.

**Comment by jaredmorgs** [comment]
Sounds cool. How would you do this?

**Comment by mojavelinux** [comment]
This is actually trickier than I first thought because we have to consider all include files. Asciidoctor does not yet provide this information via the API, though it is planned. See https://github.com/asciidoctor/asciidoctor/issues/1163. We'll need that (or otherwise we have to use a custom script, which may not be worth the effort).

**Comment by mojavelinux** [comment]
Note that right now the footer reflects the last time the site was built, which is on every commit (even if it doesn't affect the user manual).

**Comment by jaredmorgs** [comment]
My call would be to make it as part of the API, or just not do it.

On Tue, 26 Jan 2016 at 18:51 Dan Allen <notifications@github.com> wrote:

> Note that right now the footer reflects the last time the site was built,
> which is on every commit (even if it doesn't affect the user manual).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor.org/issues/130#issuecomment-174900135>
> .
>
-- 

Sent from Mobile.

**Comment by mojavelinux** [comment]
We'll mark this as blocked. Since it is linked to the issue in core, I'll know we are waiting on that change.

**Comment by mojavelinux** [comment]
This is no longer relevant in the new docs site. The last modified time is not shown, and the web server (Netlify) will report the last modified time of the page as the time the page was visited (since that's how Netlify works). Each page in the site has a "edit this page", which links to the location of the page source in the repository. By clicking "Cancel", you can find a link to see the edit history of that page.
